docs/DeveloperGuide.md

```diff
@@ -255,6 +255,27 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### \[Proposed\] Update Find Command
+
+#### Proposed Implementation
+
+Enhance the `find` command to enable searching profiles using additional attributes such as 'tags', 
+'roomNumber', and 'contactNumber'. These new features will create predicates similar to the existing 'name' feature, 
+allowing for more comprehensive search capabilities.
+
+Below is a detailed process illustration using a sequential diagram:
+
+Step 1: The user issues a `find` command followed by specific parameters, 
+for example: `t/friends n/Alex r/08-0805 p/9124 6892`.
```

Comment:
The for example is necessary, following the conventions of the DG you can directly write that the user executes `find t/friends n/Alex r/08-0805 p/9124 6892` to search for a profile with name Alex, tags friends .....